cms/models.py

```diff
@@ -809,6 +809,16 @@ class CoursePage(ProductPage):
         )
     ]
 
+    def _get_child_page_of_type(self, cls):
+        """Gets the first child page of the given type if it exists"""
+        child = self.get_children().type(cls).live().first()
+        return child.specific if child else None
+
+    @property
+    def certificate_page(self):
+        """Gets the certificate child page"""
+        return self._get_child_page_of_type(CertificatePage)
```

Comment:
We should define it into the ProductPage instead, will be reusable for programs too.